entrypoint.sh

```diff
@@ -135,20 +138,33 @@ function usesBoolean() {
 
 function useSnapshot() {
   local TIMESTAMP=`date +%Y%m%d%H%M%S`
-  local SHORT_SHA=$(echo "${GITHUB_SHA}" | cut -c1-6)
```

Comment:
SHORT_SHA used elsewhere is -c1-7, this is -c1-6. Not sure of the significance, but I figured we overflowed something somewhere and didn't want to mess with it.

Reply:
Random question. can this SHA be used in github to get back to the commit? 
For example. this work: https://github.com/TakeoffTech/Publish-Docker-Github-Action/pull/3/commits/da78917 (`da78917`)
but this does not: https://github.com/TakeoffTech/Publish-Docker-Github-Action/pull/3/commits/da7891

Reply:
I could go with `SHORT_SHA=$(git rev-parse --short HEAD)` which would extend it by a character but make it reversible. I'm not sure if integration with FLUX_TAG would be broken by that change though. Maybe Kevin would know?

Reply:
Leaving it as now is okay; we can always fix it later if it's useful/needed.  The nice thing about centralized tools.. change is less painful for decisions like this. :)   Up to you, and @scheuk .